docs/sources/installation/upgrading.md

```diff
@@ -169,3 +169,9 @@ configuration.
 If you're embedding Grafana in a `<frame>`, `<iframe>`, `<embed>` or `<object>` on a different website it will no longer work due to a new setting
 that per default instructs the browser to not allow Grafana to be embedded. Read more [here](/installation/configuration/#allow-embedding) about
 this new setting.
+
+### Session storage is no longer used
+
+In 6.2 we completely removed the session storage since we replaced the previous session implementation with an auth token.
```

Comment:
Is this really true if you're using the auth proxy or have you changed so that auth proxy now uses auth tokens?